doc/source/new/virtualtables.rst

```diff
@@ -0,0 +1,586 @@
+- Virtual tables are managed by Cassandra and a user cannot run  DDL to create new virtual tables or DML to modify existing virtual tables
+- Virtual tables are created in special keyspaces and not just any keyspace
+- All existing virtual tables use LocalPartitioner
+
+Virtual Keyspaces
+^^^^^^^^^^^^^^^^^
+
+Apache Cassandra 4.0 has added two new keyspaces for virtual tables: ``system_virtual_schema`` and ``system_views``. Run the following command to list the keyspaces.
+
+::
+
+ cqlsh> DESC KEYSPACES;
+ system_schema  system       system_distributed  system_virtual_schema
+ system_auth      system_traces       system_views
+
+The ``system_virtual_schema keyspace`` contains schema information on virtual tables. The ``system_views`` keyspace contains the actual virtual tables. The virtual keyspaces metadata is not exposed through ``DESCRIBE`` statement, which returns an error message:
+
+::
+
+ cqlsh> DESCRIBE KEYSPACE system_views
+ 'NoneType' object has no attribute 'export_for_schema'
+ cqlsh> DESCRIBE KEYSPACE system_virtual_schema;
+ 'NoneType' object has no attribute 'export_for_schema'
+ cqlsh>
+
+Virtual Table Limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Virtual tables and virtual keyspaces have some limitations such as:
+
+- Cannot alter virtual keyspaces
+- Cannot drop virtual keyspaces
+- Cannot alter virtual tables
+- Cannot drop virtual tables
+- Cannot truncate virtual tables
+- Expiring columns are not supported by virtual tables
+- Conditional updates are not supported by virtual tables
+- Cannot create tables in virtual keyspaces
+- Cannot perform any operations against virtual keyspace
+- Secondary indexes are not supported on virtual tables
+- Cannot create functions in virtual keyspaces
+- Cannot create types in virtual keyspaces
+- Materialized views are not supported on virtual tables
+- Virtual tables don't support DELETE statements
+- Cannot CREATE TRIGGER against a virtual table
+- Conditional BATCH statements cannot include mutations for virtual tables
+- Cannot include a virtual table statement in a logged batch
+- Mutations for virtual and regular tables cannot exist in the same batch
+- Conditional BATCH statements cannot include mutations for virtual tables
+- Cannot create aggregates in virtual keyspaces
+
+Listing and Describing Virtual Tables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Virtual tables in a virtual keyspace may however be listed with ``DESC TABLES``.  The ``system_views`` virtual keyspace tables include the following:
+
+::
+
+ cqlsh> USE system_views;
+ cqlsh:system_views> DESC TABLES;
+   clients    internode_inbound
+ disk_usage       sstable_tasks        caches           
+ local_writes        max_partition_size  local_reads       
+ coordinator_writes  internode_outbound  thread_pools      
+ local_scans         coordinator_reads   settings 
+
+Some of the salient virtual tables in system_views virtual keyspace are described in Table 1.
+

```

Comment:
current list is

```
cqlsh> desc tables;
...
Keyspace system_views
---------------------
sstable_tasks
disk_usage
thread_pools
local_scan_latency
coordinator_scan_latency
max_partition_size
internode_inbound
coordinator_read_latency
caches
local_read_latency
rows_per_read
clients
local_write_latency
internode_outbound
coordinator_write_latency
tombstones_per_read
settings
```

Reply:
Seems like some of those got cut off in the list above and shortened.

Reply:
Added all.